src/backend/distributed/commands/utility_hook.c

```diff
@@ -119,7 +139,7 @@ typedef struct NonMainDbDistributedStatementInfo
 
 /*
  * MarkObjectDistributedParams is used to pass parameters to the
- * MarkObjectDistributedFromNonMainDb function.
+ * MarkObjectDistributedGloballyFromNonMainDb function.
  */
```

Comment:
/*
 * DistObjectOperationParams is used to pass parameters to the
 * MarkObjectDistributedGloballyFromNonMainDb and
 * UnMarkObjectDistributedLocallyFromNonMainDb functions.
 */

Reply:
Done